<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>420</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(250 ,150,   60);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,125,50);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(250 ,150,   60);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,125,50);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(250 ,150,   60);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,125,50);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200,200,200);
  color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
 color: black; 
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>71</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200,200,200);
 color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>70</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>140</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(250 ,150,   60);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,125,50);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>70</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
 	qproperty-alignment: 'AlignVCenter | AlignRight';
	 border: 1px solid gray;
	font-size: 22px;
}

background-color : white;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200,200,200);
 color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(200,200,200);
 color: black; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: rgb(250,250,250);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
